src/page-multi-facility/FacilityRowRtValuePill.tsx

```diff
@@ -0,0 +1,111 @@
+import React from "react";
+import styled from "styled-components";
+
+import { RateOfSpreadType } from "../constants/EpidemicModel";
+import ChartTooltip from "../design-system/ChartTooltip";
+import Colors from "../design-system/Colors";
+import PillCircle from "../design-system/PillCircle";
+import {
+  TooltipContents,
+  TooltipLabel,
+  TooltipValue,
+} from "../rt-timeseries/RtTimeseries";
+
+const pillColors: {
+  [x: string]: {
+    [x: string]: string;
+  };
+} = {
+  [RateOfSpreadType.Controlled]: {
+    text: Colors.green,
+    border: Colors.teal,
+  },
+  [RateOfSpreadType.Infectious]: {
+    text: Colors.darkRed,
+    border: Colors.orange,
+  },
+  [RateOfSpreadType.Missing]: {
+    text: Colors.forest,
+    border: Colors.forest30,
+  },
```

Comment:
you changed `RateOfSpreadType` t0 a numerical enum, so these are going to be keyed like `0: {}, 1: {}, 2: {}` ... not technically wrong but kind of weird, maybe not what you want?

Reply:
Fixed.